src/Native/include/nncase/ntt/arch/x86_64/primitive_ops.h

```diff
@@ -210,7 +210,7 @@ template <> struct div<ntt::vector<float, 8>, ntt::vector<float, 8>> {
 };
 
 // floor_mod
-template <> struct floor_mod<ntt::vector<int32_t, 8>, ntt::vector<float, 8>> {
+template <> struct floor_mod<ntt::vector<int32_t, 8>, ntt::vector<int32_t, 8>> {
```

Comment:
Ditto at Line 237.

Reply:
Thanks, I have changed it as your advice.